index.html

```diff
@@ -312,7 +312,11 @@ <h4><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Floe at the
                                 <a href="http://idrc.ocadu.ca">Inclusive Design Research Centre</a>, <a href="http://ocadu.ca">OCAD University</a>.<br/>
                             </p>
                             <p>
-                                <span class="floe-hewlett-logo">Funded by a grant from <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.</span>
+                                <span class="floe-hewlett-logo">
+                                    <p>
+                                        Funded by a grant from <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.
+                                    </p>
+                                </span>
```

Comment:
These changes show no effect on the footer for smaller screens. I am unable to understand how adding p tags is related to the overflowing of images in the content part of the website or the preferences column.
1)
![Screenshot from 2020-03-04 00-21-30](https://user-images.githubusercontent.com/34926285/75809187-5d589300-5dae-11ea-8f33-f6ecb060b90f.png)
2)
![Screenshot from 2020-03-04 00-21-47](https://user-images.githubusercontent.com/34926285/75809197-634e7400-5dae-11ea-89c6-5a68fd135e94.png)

Reply:
adding paragraph tags shows no effect. And in fact you only gave indentation to this.

Reply:
Adding paragraph tag and adding indentation will show no effect.

Reply:
> adding paragraph tags shows no effect. And in fact you only gave indentation to this.

@aashimgarg This is my review of changes proposed by @thisispratt. I didn't propose to add any p tags. 

> Adding paragraph tag and adding indentation will show no effect.

That's exactly what I pointed out in the comment above and this review.